src/abstract-interpretation/data-frame/mappers/arguments.ts

```diff
@@ -38,7 +38,7 @@ export interface FunctionParameterLocation<T = undefined> {
  */
 export function escapeRegExp(text: string, allowTokens: boolean = false): string {
 	if(allowTokens) {
-		return text.replace(/[.*+?^${}()|[\]]/g, '\\$&');
+		return text.replace(/[.*+?^${}()|[\]]/g, '\\$&').replace(/\\[^stn]/g, '\\$&');
```

Comment:
This should become a "global helper function" in the future - replacing twice feels so painful

Reply:
Yes i know but is there a reason this does not happen within the first regex?